src/main/java/uk/gov/objecthash/RawValue.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.objecthash;
```

Comment:
maybe we could move all these into a `values` package so they are grouped?

Reply:
I think I'll do this in a follow up PR just in case it causes conflicts